fpm/src/fpm_sources.f90

```diff
@@ -411,6 +490,48 @@ function parse_c_source(c_filename) result(c_source)
 end function parse_c_source
 
 
+function split_n(string,delims,n,stat) result(substring)
```

Comment:
We should move these into an `fpm_string.f90` module, and eventually get these into `stdlib`.